platforms/s390x/README.md

```diff
@@ -13,25 +13,30 @@ sudo apt-get install -y librdkafka-dev
 ## Running Connect
 Compose 3.7 is supported on Ubuntu 20.04 on s390x (LinuxONE), but LinuxForHealth connect uses compose 3.9 for profile support.  Without compose 3.9 profiles, the easiest way to start connect containers on s390x is to specify the specific containers to start in your docker-compose command line using the instructions below.
 
+### Copy the s390x files
+Copy the s390x Dockerfile and docker-compose.yml to the connect directory.
+```shell
+cd connect
+cp platforms/s390x/Dockerfile .
+cp platforms/s390x/docker-compose.yml .
+```
+
```

Comment:
We need to build and run from the top-level directory, so we need to copy the s390x files there.